hash/TBBCHT.h

```diff
@@ -33,6 +33,16 @@ namespace FastHash
             }
         }
 
+        void erase(TKey idx) {
+            HashMapAccessor hashAccessor;
+            if (!m_map->find(hashAccessor, idx)) {
+                // Presumably unreachable
+                printf("m_key: %ld Presumably unreachable\n", idx);
+                return;
+            }
+            m_map.erase(hashAccessor);
+        }
+
```

Comment:
Just to confirm from before, this code is never accessed, just implemented for the sake of it right?